### PR TITLE
Prisonform

### PR DESCRIPTION
**Pull request by Aliszl**

### CODE REVIEW

File: src/components/Login.js [hunk 1]
```diff
@@ -126,10 +126,10 @@ export const Login = Form.create({ name: "normal_register" })(LoginForm);
 
 const StyledContainer = styled.div`
   display: flex;
-  justify-content: center;
+  flex-direction: column;
   align-items: center;
-  min-height: 80vh;
```
Comment (hyetigran) [inline src/components/Login.js:131]: looks like this may have been removed by accident

File: src/components/Login.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ function LoginForm(props) {
       .then(response => {
         setLoadingUser(false);
         localStorage.setItem("token", response.data.token);
-        props.history.push("/");
```
Comment (hyetigran) [inline src/components/Login.js:48]: removed by accident?

File: src/components/Login.js [hunk 3]
```diff
@@ -126,10 +126,10 @@ export const Login = Form.create({ name: "normal_register" })(LoginForm);
 
 const StyledContainer = styled.div`
   display: flex;
-  justify-content: center;
```
Comment (hyetigran) [inline src/components/Login.js:129]: removed by accident?

File: src/components/Navigation.js [hunk 4]
```diff
@@ -24,9 +24,12 @@ const Navigation = () => {
           <Link to="/addInmate">Add an Inmate </Link>
         </Menu.Item>
         <Menu.Item key="5">
-          <Link to="/login">Login </Link>
+          <Link to="/addPrison">Add a Prison </Link>
```
Comment (hyetigran) [inline src/components/Navigation.js:27]: This is okay for now to access the form but we'll need to have this link siting in the facilities page.

**Review by hyetigran — CHANGES_REQUESTED** [review]
Looks like a few lines of code from the login were accidentally removed. Could you please add them back in.

File: src/components/Login.js (hunk 1 above)
Comment (Aliszl) [reply src/components/Login.js:131]: We updated that together, not sure how it got lost.  updated

File: src/components/Login.js (hunk 2 above)
Comment (Aliszl) [reply src/components/Login.js:48]: I updated to props.history.push("/facilityList"); Is that ok? ( bearing in mind comment below about future routing)

**Review by hyetigran — APPROVED** [review]
reviewed

**Pull request merged**